src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import { useState, ChangeEvent } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number) {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
+}
+
+export const tasks = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [taskUser, setTaskUser] = useState('0');
+  const [task, setTask] = useState('');
+  const [errorTitle, setTitleError] = useState(false);
```

Comment:
same here, 
you probably want to place a string here when there is an error, thus it should be either '' or undefined.